java/com/google/devtools/moe/client/dvcs/git/GitRevisionHistory.java

```diff
@@ -89,7 +89,7 @@ public RevisionMetadata getMetadata(Revision revision) {
     }
 
     // Format: hash, author, fullAuthor, date, parents, full commit message (subject and body)
-    String format = Joiner.on(LOG_DELIMITER).join("%H", "%an", "%aN <%cE>", "%ad", "%P", "%B");
+    String format = Joiner.on(LOG_DELIMITER).join("%H", "%an", "%aN <%aE>", "%ad", "%P", "%B");
```

Comment:
This only seems to run as part of a `git log` execution. I'm confused: how does this fix your overall issue?